### DISCUSSION TITLE
v2.5.10

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#178] [Announcements]
- Fixed MoviesDrive, MultiMovies, ExtraMoves, NetFlixMirror and HdHub4U.
- Improved Extractors.

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/v2.5.10'>v2.5.10</a>.</em>

**Comment by Reddy5070** [comment]
Thanks for update
